addons/web/static/src/legacy/scss/form_view.scss

```diff
@@ -637,7 +637,7 @@ $o-form-label-margin-right: 0px;
         margin: 1.2em ($grid-gutter-width * .5) .8em 0;
 
         &:not(:empty) {
-            box-shadow: 0 1px 0 rgba($body-color, .05);
+            box-shadow: 0 1px 0 $o-form-separator-color;
```

Comment:
I guess this should be also in non-legacy form view :wink:

Reply:
yep, thanks :)